clang/lib/Lex/PPDirectives.cpp

```diff
@@ -177,6 +177,26 @@ static bool isLanguageDefinedBuiltin(const SourceManager &SourceMgr,
   return false;
 }
 
+static bool isReservedAttrName(Preprocessor &PP, IdentifierInfo *II) {
```

Comment:
What should the behavior be with `-nostdlib`? Should these actually be reserved in this case (given that the restriction is under "library-wide requirements")?

Reply:
`-nostdlib` doesn't imply that you don't have a standard library, it just implies that you don't use the system-provided one, so I'd say it shouldn't influence the diagnostic. If you don't care you can just disable the diagnostic still, right?